src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,8 @@
 package mate.academy.service;
 
-public class AuthenticationService {
+import mate.academy.model.User;
+
+public class AuthenticationService extends UserService {
```

Comment:
in order to use `findByEmail()` method you need to create `UserService` object, not extend from `UserService`